src/ray/object_manager/pull_manager.cc

```diff
@@ -782,4 +782,9 @@ std::string PullManager::DebugString() const {
   return result.str();
 }
 
+void PullManager::SetOutOfDisk(const ObjectID &object_id) {
+  RAY_LOG(DEBUG) << "Pull of object failed due to out of disk: " << object_id;
+  fail_pull_request_(object_id, rpc::ErrorType::OBJECT_FETCH_OUT_OF_DISK);
```

Comment:
(I guess you do), do we test this from the python test?

Reply:
yup, any pulled object goes through this code path. (task arguments/ray get)